src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java

```diff
@@ -112,4 +122,19 @@ public AddClubPopUpStepOne clickCenterDropdown(){
         return this;
     }
 
+    @Step("Click on the dropdown with a list of centers")
+    public AddClubPopUpStepOne clickEditCenterDropdown(){
+        centerSelectedInEditClub.click();
+        return this;
+    }
+
+    @Step("Select new center for club")
+    public AddClubPopUpStepOne changeCenterForClub(String value) {
```

Comment:
duplicate method, we have selectCenter for select center

Reply:
Removed the duplicate, updated the selectCenter method